src/app/rpmostree-builtin-status.cxx

```diff
@@ -688,6 +688,27 @@ print_one_deployment (RPMOSTreeSysroot *sysroot_proxy, GVariant *child, gboolean
     }
   else
     g_print ("%s", checksum);
+
+  /* Get deployment index */
+  int index = 0;
+  GVariant *deployments = rpmostree_sysroot_get_deployments (sysroot_proxy);
+  GVariantIter iter;
+  g_variant_iter_init (&iter, deployments);
+  while (true)
```

Comment:
Instead of this, let's just pass the index to `print_one_deployment`? We can change the `gboolean first` to be a e.g. `guint index` and then things which depended on `first` can just check for `index == 0`.

Reply:
Sounds reasonable, I added that. I decided to add `gboolean first = (index == 0);` to `print_one_deployment` instead. I think it reads much nicer than spraying `(index [!=]= 0)` throughout the code. Hope that's ok!